File: C/Estudos_C/EntradaDeDados.c
#include <stdio.h>
#include <string.h>

int main() {
    double altura, salario;
    int idade;
    char nome[50];

    printf("\nDigite o primeiro nome: ");
    scanf(" %s", nome);

    printf("\nDigite a idade: ");
    scanf(" %d", &idade);

    printf("\nDigite sua altura: ");
    scanf(" %lf", &altura);

    printf("\nDigite seu salario: ");
    scanf(" %lf", &salario);

    printf("%s tem %d anos, %.2lf Metros e ganha $ %.3lf  ", nome, idade, altura, salario);


    return 0;
}

File: Python/Estudos_python/EntradaDeDados.py
nome1: str
nome2: str
salario1: float
salario2: float
genero: str
idade:int

nome1 = str(input("digite o primeiro nome: "))
salario1 = float(input("digite o primeiro salario: "))
print("//////////////////////////////////////////")
nome2 = str(input("digite o segundo nome: "))
salario2 = float(input("digite o segundo salario: "))
print("//////////////////////////////////////////")
genero = str(input("Digite um genero: "))
idade = int(input("digite sua idade: "))
print("//////////////////////////////////////////")
print(" ")
print(f"a/o {nome1} ganha um salario de {salario1}")
print(f"a/o {nome2} ganha um salario de {salario2}")
print(f"genero: {genero}")
print(f"idade: {idade}")